mmv1/products/compute/api.yaml

```diff
@@ -6129,6 +6129,13 @@ objects:
                   Defines whether the instance is preemptible. This can only be set
                   during instance creation, it cannot be set or changed after the
                   instance has been created.
+              - !ruby/object:Api::Type::Enum
+                name: 'maintenanceInterval'
+                description: |
+                  Specifies the frequency of planned maintenance events.
+                values:
+                - :PERIODIC
+                min_version: "beta"
```

Comment:
It looks like there is a conflict with the main branch now. You can delete the code block.